src/XRP/issued-currency-client.ts

```diff
@@ -379,4 +385,163 @@ export default class IssuedCurrencyClient {
+   *       - therefore this method is incapable of initiating a cross-currency payment.
+   *
+   *
+   * Case - Creating an issued currency:
+   *      The sending address of the transaction is the same address as the issuer of the `Amount` field.
+   *
+   *      No SendMax field is required because the Transfer Fee doesn't apply to transactions in which
+   *              the originator or the destination address is the same as the issuer.
+   *               (you don't get charged to handle your own money, or let your customers redeem their money with you.)
+   *
+   *      ... does the destination address need to have a trustline established to the sender/issuer?
+   *
+   *
+   * Case - dispensing an issued currency from an operational address (https://xrpl.org/issuing-and-operational-addresses.html):
+   *      The sending address (an operational address) is not the same address as the issuer of the `Amount` field
+   *
+   *      Issuer needs to allow rippling (https://xrpl.org/rippling.html).
+   *
+   *      A SendMax field is required here, because the transfer fee will apply as the sender of this txn is not the issuer.
+   *
+   *
+   * Case - sending an issued currency payment from one XRPL address to another:
+   *      Identical to dispensing an issued currency from an operational address (which is just making an issued currency payment)
+   *
+   *
+   * Case - redeeming an issued currency with the issuer:
+   *      The destination address is the same address as the issuer of the `Amount` field.
+   *
+   *      No SendMax field is required because the Transfer Fee doesn't apply (see first Case above)
+   *==================================================================================================================================
+   *
+   * @param sender The Wallet from which issued currency will be sent, and that will sign the transaction.
+   * @param destinationAddress The destination address (recipient) for the payment, encoded as an X-address (see https://xrpaddress.info/).
+   * @param currency The three-letter currency code of the issued currency being sent.
+   * @param issuer The issuing address of the issued currency being sent.
+   * @param value The amount of issued currency to send.
+   * @param transferFee Optional - can be omitted if sender address or destinationAddress are the same address as the issuer.
+   *                    The transfer fee associated with the issuing account, expressed as a percentage. (i.e. a value of .5 indicates a 0.5% transfer fee).
+   */
+  // TODO: (acorso) make this private if/when incorporated into higher level convenience methods
+  // TODO: (acorso) consider using an object for potentially long list of params
+  public async sendIssuedCurrency(
+    sender: Wallet,
+    destinationAddress: string,
+    currency: string,
+    issuer: string,
+    value: string,
```

Comment:
`amount`? 

also i noticed `send` is a `BigInteger | number | string` - do you expect us to offer an interface like that here as well?

Reply:
that's a good question... I suppose we could allow for working with any of those types, even though they'll be converted to a string for submission to the network.  I'll make an asana ticket for this for the moment: https://app.asana.com/0/1190819649053561/1198956705900887/f